internal/revision/revision.go

```diff
@@ -24,17 +24,17 @@ func FindLatestHash(pwd string) (string, error) {
 
 	/// HEAD points to a commit in particular.
 	if len(headData) > 0 {
-		return headData, nil
+		return strings.ReplaceAll(headData, "\n", ""), nil
```

Comment:
What is this change for? Do we have a test for it?

Reply:
because the hash, is being returned with a \n after the string. so when i added to the header in the request, the request it throws an error. i will add an extra test to cover it